Refits the derelict laboratory into break room and office

babydoll opened the pull request:
Thread and poll - option B has won.
https://baystation12.net/forums/threads/poll-derilect-laboratory-refitting.5386/
![](https://s1.postimg.org/73iltwumq7/1507061813.png)


Fixes #18766 - adds honey extractors to xenoflora and hydroponics. Also makes the machine solid and stationary

sabiram commented:
If this is approved, I'll begin the dark task of resolving conflicts

Crushtoe commented:
Not a fan of removing an open space people can renovate as they wish.

babydoll commented:
Multiple people have asked for an office for the SR - this is the best place for it, and it's not actually 'used' by the company or the ship in the game world, since it's been torn down and abandoned. Seems a bit silly to have such a room in the state of the art research division.

I know there are people (myself included) who like to renovate this room. The window between the office and the break room isn't electrified so renovations are still possible without special equipment. There's very little I placed that can't be torn down with relative ease. Of course, you'll now need permission from the person whose office you're tearing apart.

AtomicWorm commented:
Odd that you go through the break room to reach the office, though.

Crushtoe commented:
Does a Senior Researcher need an office? Senior Engineers don't have one, Physicians don't have one which used to be the Senior Doctor, does anyone but department heads really have to have an office?

babydoll commented:
This was unused space. Someone is presumably paying for it. NT is renting the space, or Sol is paying NT to occupy the space. In either case, this is valuable real estate right in the research wing that is going completely unused. I kept the shell of the room so it's like it really was refitted recently, and this is what they fit into the shell.

Nobody does their primary job in their office, except maybe the XO. And yet all those people have offices. The SR is (I think) the acting-RD when one isn't present, so they have some kind of claim to needing an office, as an executive. 

You'll have to ask the people who wanted this why **they** wanted it. I saw a request for it and thought I would help.

ghost commented:
Does the blast doors in the window in the new "break room" close alongside the other blast doors during a lockdown now?

ghost commented:
> Not a fan of removing an open space people can renovate as they wish.

IMO, materials intended for research are wasted on renovating a gimmick room. Robotics isn't even part of science anymore, so why would there be a room, even if abandoned, at all, if it's just going to be unused?

>Odd that you go through the break room to reach the office, though.

![image](https://user-images.githubusercontent.com/31956636/31113779-1d3554c0-a81b-11e7-8cf4-fd6c5067939c.png)
senior_researcher_ensuring_break_room_isnt_used_for_slacking_off.png ¯\\\_(ツ)_/¯
As a second solution, couldn't we just remove/make the teleporter room on Deck 1 more compact?

> Does a Senior Researcher need an office? Senior Engineers don't have one, Physicians don't have one which used to be the Senior Doctor, does anyone but department heads really have to have an office?

IMO, technically, the Senior Engineer's office would/could be telecomms, and the Physicians office would/could be the examination room. And then there's the point that was brought up that it's NT and they are free to do whatever they wish with their department (to a certain degree, of course.)

babydoll commented:
The blast doors are tied to the same 'Biohazard' network in the security office, yes.

Aetsuki commented:
Senior Researcher does need an office in my opinion. He's the fill in for the RD if there isn't one and as NT staff he's already liable to have to do paperwork. People don't renovate the robotics lab that much because any work they put into it is gone the next round. People'll do it once or twice, but beyond that it's hardly used. Maybe a cultist builds and forgets a crystal in there.

Techhead0 commented:
I'd rather see an all-NT-use bullpen-style office if NT people have to do paperwork and such. And if SR is filling in for RD, then she can talk to the CO about using the RD'office.

babydoll commented:
Thread and poll
https://baystation12.net/forums/threads/poll-derilect-laboratory-refitting.5386/

babydoll committed:
Refit the dead lab in research into breakroom and workstations

Refits the derelict laboratory in research into a shared space with
tables and stools on one side with snack machines and computer desks
on the other side for various activities.

babydoll commented:
PR updated with option B. Removed now-unused access and areas.

babydoll commented:
Changelog added too

comma commented:
Nyeh Nyeh.
For conflict you can just use your version and put a machinery/pager/medical somewhere in med lobby.

PsiOmegaDelta commented:
Conflicts.

babydoll committed:
Merge branch 'dev' of https://github.com/Baystation12/Baystation12 into AWNP-optionB

# Conflicts:
#	maps/torch/torch-4.dmm

babydoll commented:
Conflicts resolved
@Techhead0

mustafakalash commented on code/modules/hydroponics/beekeeping/beehive.dm:
```diff
@@ -156,9 +156,11 @@
 
```
Beehive changes?

Techhead0 replied:
Listed in the PR OP.

mustafakalash replied:
Weird. Oh well.

mustafakalash commented on html/changelogs/sabiram - breaky.yml:
```diff
@@ -0,0 +1,6 @@
+author: Unknown
```
Author.

Techhead0 replied:
If someone opts to go uncredited, that's their option. It's also why we changed the default to "Unknown" instead.
